src/alert/alert.ts

```diff
@@ -60,11 +60,16 @@ export class NgbAlert {
  * Alerts that can be dismissed without any additional code.
  */
 @Directive({selector: 'template[ngbAlert]'})
-export class NgbDismissibleAlert implements OnInit, OnDestroy {
+export class NgbSelfClosingAlert implements OnInit, OnDestroy {
   private _popupService: PopupService<NgbAlert>;
   private _timeout;
 
   /**
+   * A flag indicating if the alert can be dismissed (closed) by a user. If this flag is set, a close button (in a
+   * form of a cross) will be displayed.
```

Comment:
`&times;`? "times"?  "x"?  just a stupid nit here on my part, but i think of "cross" as a plus.